src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -3447,12 +3447,16 @@ def server_create(
     if enable_ad_only_auth:
         ad_only = True
 
+    tenant_id = None
```

Comment:
NIT: not pythonic~ LOL
tenant_id=_get_tenant_id() if external_admin_name is not None else None

Reply:
azdev style sql
azdev linter sql 

pass :)

Reply:
Style asked me to switch to `is not None` :
![image](https://user-images.githubusercontent.com/70013163/123384668-88d63280-d549-11eb-9f99-321e2f2799d6.png)

This is the exact reason I left Data Science track: 
tenant_id=_get_tenant_id() if external_admin_name is not None else None does not even seem like code, I mean it's like writing a story. Not readable at all.

Reply:
LOL~ that's python~ LMAO

but yes, if any logic is used twice, it should be abstracted as a method~